app/models/comment_model.rb

```diff
@@ -0,0 +1,9 @@
+class CommentModel < ApplicationRecord
```

Comment:
- [ ] Please rename your models' names to `Comment`, `User`, `Post`, etc. This is because the Ruby name conventions make the associations relative to the models' names. For example, the `CommentModel` would be in the associations `has_many :comment_models`, Please fix the names in all other models.

Reply:
Needs attention!